go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/GoogleCloudPlatform/golang-samples
 go 1.11
 
 require (
-	cloud.google.com/go v0.49.0
+	cloud.google.com/go v0.49.1-0.20191216225856-71c1cb005d22
```

Comment:
Not sure the best way to handle this. Secret Manager is in the main repo, but not in a tagged version yet.